pkgdown/_pkgdown.yml

```diff
@@ -25,6 +33,7 @@ reference:
   - dm_get_all_pks
```

Comment:
For functions that return a data frame we also need to check if the output column names make sense.

Reply:
outsourced this comment to issue #230